examples/rust/Earthfile

```diff
@@ -0,0 +1,79 @@
+VERSION 0.7
+
+# cspell: words TARGETARCH USERARCH toolsets
+
+# Set up our target toolchains, and copy our files.
+builder:
+    FROM ./../../earthly/rust+rust-base
+
+    COPY --dir "*" .
```

Comment:
Use CP_SRC and only copy files that affect the rust build itself. Exclude anything that does not, such as Readme.md
or any other files not part of the rust build.